charts/newrelic-infra-operator/values.yaml

```diff
@@ -113,7 +113,7 @@ config:
 ## Image of the infrastructure agent to be injected.
       image:
         repository: newrelic/infrastructure-k8s
-        tag: 2.8.1-unprivileged
```

Comment:
can you bump as well the info in the docs https://github.com/newrelic/helm-charts/blob/b01b6fe6cb622377f3dda61aca102cc13a87e0dc/charts/newrelic-infra-operator/README.md?plain=1#L53

Reply:
done